### PR TITLE
[Term Entry] Kotlin Math .exp()

### LINKED ISSUE #3523: [Term Entry] Kotlin Math .exp()
### Reason/inspiration (optional)

We would like a new term entry on the `.exp()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/exp/exp.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by vaisrinivasan**
### Description

I created the exp.md in Kotlin > concepts > math-methods > terms directory.
Closes #3523 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by vaisrinivasan** [comment]
Hi. Need some help with the markdown part (The initial one). 
`---
Title: 'sqrt()'
Description: 'Calculates the square root of a given number.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Math'
  - 'Functions'
    CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---`
Whichever solution I try, it doesn't come as a table like given in the template and hence the test is failing.

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:3]:
```suggestion
Description: 'Returns e raised to the power of a given number.'
```

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:15]:
```suggestion
The **`exp()`** method in Kotlin's `math` class is used to calculate Euler's number *e* raised to the power of a given number.
```

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:23]:
```suggestion
- `number`: A number of type `Double` or `Float`. The method returns a number, of the same type given.
```

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:25]:
```suggestion
```

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:29]:
```suggestion
This example demonstrates a basic implementation of the `exp()` method.
```

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:36]:
```suggestion
    println("Euler's number e raised to the power of $number is $ePowNumber.")
```

File: content/kotlin/concepts/math-methods/terms/exp/exp.md
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/exp/exp.md:43]:
```suggestion
Euler's number e raised to the power of 3.0 is 20.085536923187668.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@vaisrinivasan I've completed my initial review. Please update the entry, and tag me when you've made the changes. Thanks

**Comment by vaisrinivasan** [comment]
@caupolicandiaz : I've addressed the review comments and all the checks are passing. Please re-review.
I haven't done the below checks as the review is already happening in this PR. Let me know if I've to push from a branch named after the changes I'm making and I'll be happy to accommodate the same.

 I have confirmed my changes are not being pushed from my forked main branch.
 I have confirmed that I'm pushing from a new branch named after the changes I'm making.

**Review by caupolicandiaz — APPROVED** [review]
@vaisrinivasan This is ready for a second review.

**Comment by vaisrinivasan** [comment]
Hi. Can I get a second review for this? Thanks.

**Comment by caupolicandiaz** [comment]
> Hi. Can I get a second review for this? Thanks.

Please be patient, there are many PRs and few maintainers

**Review by SSwiniarski — APPROVED** [review]
@vaisrinivasan Looks good.

**Pull request merged**
